### ISSUE TITLE
Choose language by dropdown menu

### ISSUE THREAD

**Issue by punkr0ck** [fix/Lisk-Lite-Client#39]
It should be possible to choose the language by dropdown menu and not by clicking through the languages.

**Comment by fix** [comment]
yes, was not sure what would be the most simple. If I put a select, there is a UX need for a validation button.
